autoPyTorch/data/tabular_feature_validator.py

```diff
@@ -63,18 +65,25 @@ def get_tabular_preprocessors() -> Dict[str, List[BaseEstimator]]:
     preprocessors['numerical'] = list()
     preprocessors['categorical'] = list()
 
-    preprocessors['categorical'].append(OneHotEncoder(
-        categories='auto',
-        sparse=False,
-        handle_unknown='ignore'))
-    preprocessors['numerical'].append(SimpleImputer(strategy='median',
-                                                    copy=False))
-    preprocessors['numerical'].append(StandardScaler(with_mean=True, with_std=True, copy=False))
+    preprocessors['categorical'].append(
+        OneHotEncoder(
+            categories='auto',
+            sparse=False,
+            handle_unknown='ignore',
+        )
+    )
+    preprocessors['numerical'].append(
```

Comment:
are we going to scale inside the pipeline?

Reply:
Yes, in the future yes.

Reply:
but do we not ned it for our experiments?

Reply:
Ups, now I see what you mean. It was wrongly deleted during rebase, I just added it.